requirements.txt

```diff
@@ -1,10 +1,11 @@
 --extra-index-url https://pypi.anaconda.org/OpenEye/simple
-OpenEye-toolkits == 2020.2.2
+OpenEye-toolkits >= 2020.2.2
```

Comment:
I'm not sure if this is a good idea or not...I know I've run into issues with OpenEye versioning before. I think this will be fine, but want to make a note of it in case we see issues down the line.

For example, our weekly update code specifically uses 2020.2.2: https://github.com/rcsb/weekly-update-workflow/blob/1c72b549e5dcef185b6d3aaf3852bcdb92175482/requirements.txt#L7

Reply:
Pinning to a specific version makes sense then, especially if our other projects are pinning as well. I've pushed a commit to revert this change.

Reply:
OK sounds good, though we do have projects that specify the latest version too...and they seem to build and run properly...sometimes (https://github.com/rcsb/py-rcsb_exdb/blob/master/requirements.txt). Anyway, pinning it should keep it safe.